staging/src/k8s.io/client-go/scale/client.go

```diff
@@ -138,7 +138,7 @@ func (c *namespacedScaleClient) Get(resource schema.GroupResource, name string)
 		SubResource("scale").
 		Do()
 	if err := result.Error(); err != nil {
-		return nil, fmt.Errorf("could not fetch the scale for %s %s: %v", resource.String(), name, err)
```

Comment:
And this wrapping made it impossible to check the error types.  I really dislike golang errors